catalyst_voices/packages/internal/catalyst_voices_repositories/lib/src/dto/document/document_ref_dto.dart

```diff
@@ -38,6 +62,25 @@ final class DocumentRefDto {
     );
   }
 
+  @override
+  int get hashCode => Object.hash(id, ver, type);
+
+  // not using Equatable because it's messing up JsonSerializable with props getter
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is DocumentRefDto &&
+          runtimeType == other.runtimeType &&
+          id == other.id &&
+          ver == other.ver &&
+          type == other.type;
+
+  String toFlatten() {
```

Comment:
`toFlatten` sounds weird, usually it's called `flatten` or `flattened`

```suggestion
  String flatten() {
```